src/dblib_stubs.c

```diff
@@ -30,13 +30,50 @@
 #include <caml/fail.h>
 #include <caml/callback.h>
 #include <caml/custom.h>
+#include <caml/threads.h>
 
 /* OCaml severity.  Keep in sync with OCaml [Dblib]. */
 #define SEVERITY_PROGRAM Val_int(6)
 #define SEVERITY_RESOURCE Val_int(7)
 #define SEVERITY_FATAL Val_int(9)
 #define SEVERITY_CONSISTENCY Val_int(10)
 
+static int release_lock(DBPROCESS *dbproc)
```

Comment:
Wouldn't `freetds_release_runtime_system` be clearer?

Reply:
Makes sense